File: psets/pset6/mario.py
#!/usr/bin/python3

import sys
import cs50

def main():
    while True:
        print("Height: ")
        row=cs50.get_int()
        if row>0 and row<23:
            break
    
    for i in range(row):
        for j in range(row):
            if j >= row - i - 1:
                print("#", end = "")
            else:
                print(" ", end = "")
        print("  ", end = "")
        for j in range(row):
            if j <= i:
                print("#", end = "")
            else:
                print(" ", end = "")
        print("\n", end = "")

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int row , i, j;
    do {
        printf("Height: ");
        row=get_int();
    } while( row < 0 || row > 23 );
    
    for(i = 0; i < row; i++) {
        for( j = 0; j < row; j++){
            if(j >= row - i - 1)
                printf("#");
            else
                printf(" ");
        }
        printf("  ");
        for( j = 0; j < row; j++){
            if(j <= i)
                printf("#");
            else
                printf(" ");
        }
        printf("\n");
    }
    
}